.travis.yml

```diff
@@ -9,11 +9,13 @@ services:
 matrix:
     include:
         - language: python
+          name: "Documentation"
           python: 3.6
           cache: pip
           script:
             - scripts/ci-docs
         - language: python
+          name: "Python (lint + test)"
```

Comment:
Rename this to "Tests + Python linting" (or however you want to spell it) and it'll be accurate.